### PR TITLE
Django18

### PR DESCRIPTION
**Pull request by emord**
Open for tests out of curiosity

### CODE REVIEW

File: corehq/apps/style/forms/widgets.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from django import forms
 from django.forms.fields import MultiValueField, CharField
 from django.forms.util import flatatt
-from django.forms.widgets import CheckboxInput, HiddenInput, Input, RadioSelect, RadioFieldRenderer, RadioInput, TextInput, MultiWidget
+from django.forms.widgets import CheckboxInput, HiddenInput, Input, RadioSelect, RadioFieldRenderer, RadioChoiceInput, TextInput, MultiWidget
```
Comment (dimagimon) [inline corehq/apps/style/forms/widgets.py]: F401 'HiddenInput' imported but unused

Comment (dimagimon) [inline corehq/apps/style/forms/widgets.py]: E501 line too long (141 > 115 characters)

File: corehq/apps/style/forms/widgets.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ def render(self, name, value, attrs=None):
         return mark_safe(u'<label class="checkbox"><input%s /> %s</label>' %
                          (flatatt(final_attrs), self.inline_label))
 
-class BootstrapRadioInput(RadioInput):
+class BootstrapRadioInput(RadioChoiceInput):
```
Comment (dimagimon) [inline corehq/apps/style/forms/widgets.py]: E302 expected 2 blank lines, found 1

**Comment by dannyroberts** [comment]
ha!

File: corehq/apps/style/forms/widgets.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 from django import forms
 from django.forms.fields import MultiValueField, CharField
-from django.forms.utils import flatatt
-from django.forms.widgets import CheckboxInput, HiddenInput, Input, RadioSelect, RadioFieldRenderer, RadioInput, TextInput, MultiWidget
+from django.forms.util import flatatt
+from django.forms.widgets import CheckboxInput, HiddenInput, Input, RadioSelect, RadioFieldRenderer, RadioChoiceInput, TextInput, MultiWidget
```
Comment (dimagimon) [inline corehq/apps/style/forms/widgets.py]: F401 'HiddenInput' imported but unused

Comment (dimagimon) [inline corehq/apps/style/forms/widgets.py]: E501 line too long (141 > 115 characters)

**Pull request closed without merging**